robottelo/constants.py

```diff
@@ -962,3 +962,60 @@
     u'alpha', u'numeric', u'alphanumeric',
     u'latin1', u'utf8', u'cjk', u'html'
 ]
+
+
+# All UI crud classes listed here to allow dynamical import
+# import import_string; import_string('robottelo.ui.{0}'.format(item))
+UI_CRUD = [
```

Comment:
I'll do that @elyezer this is the same question @cswiii asked when I made the demo.
I am thinking in implement that in a separate PR.